src/sonic-frr/patch/series

```diff
@@ -65,3 +65,4 @@
 0082-Revert-bgpd-upon-if-event-evaluate-bnc-with-matching.patch
 0083-staticd-add-cli-to-support-steering-of-ipv4-traffic-over-srv6-sid-list.patch
 0084-lib-Return-duplicate-prefix-list-entry-test.patch
+0084-This-error-happens-when-we-try-to-write-to-a-socket.patch
```

Comment:
>0084 [](http://example.com/codeflow?start=0&length=4)

Could you use non occupied number? okay to skip some.

Reply:
Done